js/civs.js

```diff
@@ -1,301 +1,396 @@
-            unique([CONQUISTADOR, ELITE_CONQUISTADOR, INQUISITION, SUPREMACY]);
-            break;
-        case "Tatars":
-            enable([], [STEPPE_LANCER, ELITE_STEPPE_LANCER], []);
-            disable([KEEP],
-                [ARBALESTER, CHAMPION, HALBERDIER, PALADIN, SIEGE_ONAGER, BOMBARD_CANNON, HEAVY_DEMO_SHIP],
-                [CHAIN_MAIL_ARMOR, PLATE_MAIL_ARMOR, SHIPWRIGHT, ARCHITECTURE, KEEP, ARROWSLITS,
-                    HOARDINGS, REDEMPTION, HERESY, SANCTITY, FAITH, THEOCRACY, STONE_SHAFT_MINING, TWO_MAN_SAW]);
-            unique([KESHIK, ELITE_KESHIK, SILK_ARMOR, TIMURID_SIEGECRAFT]);
-            break;
-        case "Teutons":
-            disable([],
-                [ARBALESTER, HEAVY_CAV_ARCHER, LIGHT_CAVALRY, HUSSAR, CAMEL_RIDER, HEAVY_CAMEL_RIDER,
-                    ELITE_CANNON_GALLEON, SIEGE_RAM],
-                [THUMB_RING, PARTHIAN_TACTICS, HUSBANDRY, DRY_DOCK, SHIPWRIGHT, BRACER,
-                    ARCHITECTURE, GOLD_SHAFT_MINING]);
-            unique([TEUTONIC_KNIGHT, ELITE_TEUTONIC_KNIGHT, IRONCLAD, CRENELLATIONS]);
-            break;
-        case "Turks":
-            disable([],
-                [ARBALESTER, ELITE_SKIRMISHER, PIKEMAN, HALBERDIER, PALADIN, FAST_FIRE_SHIP, ONAGER,
-                    SIEGE_ONAGER],
-                [HERBAL_MEDICINE, ILLUMINATION, BLOCK_PRINTING, STONE_SHAFT_MINING, CROP_ROTATION,
-                    SIEGE_ENGINEERS]);
-            unique([JANISSARY, ELITE_JANISSARY, SIPAHI, ARTILLERY], MONK_PREFIX_AFRICAN);
-            break;
-        case "Vietnamese":
-            enable([], [BATTLE_ELEPHANT, ELITE_BATTLE_ELEPHANT, IMPERIAL_SKIRMISHER], []);
-            disable([],
-                [HAND_CANNONEER, HUSSAR, PALADIN, CAMEL_RIDER, HEAVY_CAMEL_RIDER, FAST_FIRE_SHIP, SIEGE_RAM,
-                    SIEGE_ONAGER, HEAVY_SCORPION,],
-                [PARTHIAN_TACTICS, SHIPWRIGHT, REDEMPTION, HERESY, FERVOR, BLAST_FURNACE,
-                    MASONRY, ARCHITECTURE, GOLD_SHAFT_MINING]);
-            unique([RATTAN_ARCHER, ELITE_RATTAN_ARCHER, CHATRAS, PAPER_MONEY], MONK_PREFIX_ASIAN);
-            break;
-        case "Vikings":
-            enable([], [LONGBOAT, ELITE_LONGBOAT], []);
-            disable([KEEP, BOMBARD_TOWER],
-                [HAND_CANNONEER, HEAVY_CAV_ARCHER, HALBERDIER, HUSSAR, CAMEL_RIDER, HEAVY_CAMEL_RIDER, PALADIN,
-                    FIRE_GALLEY, FIRE_SHIP, FAST_FIRE_SHIP, SIEGE_ONAGER, BOMBARD_CANNON],
-                [PARTHIAN_TACTICS, BLOODLINES, HUSBANDRY, SHIPWRIGHT, KEEP, BOMBARD_TOWER, REDEMPTION,
-                    HERBAL_MEDICINE, SANCTITY, ILLUMINATION, THEOCRACY, PLATE_BARDING_ARMOR,
-                    STONE_SHAFT_MINING, GUILDS]);
-            unique([BERSERK, ELITE_BERSERK, CHIEFTAINS, BERSERKERGANG]);
-            break;
-    }
+
+    let selectedCiv = civsConfig[name];
+
+    let enabled = selectedCiv.enabled;
+    let disabled = selectedCiv.disabled;
+    if (selectedCiv.disableHorses) disableHorses(tree);
```

Comment:
I know this project does not have written style guidelines, so this expectation has not been obvious, but please use curly braces on all `if` statements.

```suggestion
    if (selectedCiv.disableHorses) {
        disableHorses(tree);
    }
```